modules/ROOT/pages/branded-emails.adoc

```diff
@@ -36,7 +36,7 @@ To enable custom email notifications, follow these steps.
 . Set *Polling Interval for External Change Data Capture (seconds)*.
 +
 The default is 1800 and the minimum is 300.
-. Navigate to *Setup*, use *Quick Find* to go to *Change Data Capture*, navigate to *Entities* > *Change Manually* > *Available Entities*, and move *AnypointContracts* to the *Selected Entities* list.
+. Navigate to *Setup*, use *Quick Find* to go to *Change Data Capture*, navigate to *Entities* > *Change Manually* > *Available Entities*, and move *AnypointContracts* to the *Selected Entities* list, which enables polling on this object.
```

Comment:
this phrasing is a little off "use *Quick Find* to go to *Change Data Capture*" , esp when other commands say "navigate to"

also maybe this should more than one step?

Reply:
You're right, this would be better as separate steps. Fixed.